pkg/build/node/const.go

```diff
@@ -138,7 +138,10 @@ data:
       "name": "kindOfBridge",
       "plugins": [
         {
-          "type": "ptp",
+          "type": "bridge",
+          "ipMasq": true,
```

Comment:
we shouldnt set ipmasq to true
>pods on a node can communicate with all pods on all nodes without NAT

https://kubernetes.io/docs/concepts/cluster-administration/networking/

Reply:
pretty sure this is a normal thing to do https://kubernetes.io/docs/tasks/administer-cluster/ip-masq-agent/, will dig more in a bit.

Reply:
we use the ip masq agent currently